src/components/Search/index.tsx

```diff
@@ -156,6 +165,14 @@ function Search({queryJSON, onSearchListScroll, contentContainerStyle}: SearchPr
 
     const searchResults = currentSearchResults?.data ? currentSearchResults : lastSearchResultsRef.current;
 
+    const {newSearchResultKey, handleSelectionListScroll} = useSearchHighlightAndScroll({
+        searchResults,
+        transactions,
+        previousTransactions,
+        queryJSON,
+        offset,
```

Comment:
We probably should have added a `offsetForRecentChat` here as well which would have been always `0`. This later caused https://github.com/Expensify/App/issues/54785